### DISCUSSION TITLE
Is there a guide available for using Manual Mesh leveling?

### DISCUSSION THREAD

**Discussion by cdenbrook** [mriscoc/Ender3V2S1#1347] [Q&A]
I have install version Ender3V2-422-MM-MPC-20240125.bin on my Ender 3 v2 and have done a basic bed level.  Prior to installing this version, I have been using JyersUI. In his ManualMesh version, there is a microstep up and down control to fine tune the Z position for each mesh point.  How do you do the comparable step in MRiscoC firmware?  There is a "Move Z" control that varies from -1.0 to +1.0.  Is this what you use? and if so, where should Z be before starting?

**Comment by mriscoc** [comment]
> Move Z" control that varies from -1.0 to +1.0.

If you need a bigger range you need to edit the firmware, but 1.0 mm for a mesh compensation is a lot, typically values are not bigger than 0.2 mm Maybe your need to do a z-offset first, then a bed tramming and finally a manual mesh leveling.
